contrib/bcc/patches/doc-loc.patch

```diff
@@ -0,0 +1,18 @@
+--- a/man/man8/CMakeLists.txt
++++ b/man/man8/CMakeLists.txt
+@@ -9,4 +9,4 @@
+   list(APPEND GZFILES "${CMAKE_CURRENT_BINARY_DIR}/${NAME}.gz")
+ endforeach()
+ add_custom_target(man ALL DEPENDS ${GZFILES})
+-install(FILES ${GZFILES} DESTINATION share/bcc/man/man8)
++install(FILES ${GZFILES} DESTINATION share/man/man8)
+--- a/tools/CMakeLists.txt
++++ b/tools/CMakeLists.txt
+@@ -21,6 +21,6 @@
+   endif()
+ endforeach()
+ install(FILES ${C_FILES} DESTINATION share/bcc/tools)
+-install(FILES ${TXT_FILES} DESTINATION share/bcc/tools/doc)
```

Comment:
/usr/share/bcc/tools has these:  

```
argdist*
bashreadline*
bindsnoop*
biolatency*
biolatpcts*
biopattern*
biosnoop*
biotop*
bitesize*
bpflist*
btrfsdist*
btrfsslower*
cachestat*
cachetop*
capable*
cobjnew*
compactsnoop*
cpudist*
cpuunclaimed*
criticalstat*
dbslower*
dbstat*
dcsnoop*
dcstat*
deadlock*
deadlock.c
dirtop*
doc/
drsnoop*
execsnoop*
exitsnoop*
ext4dist*
ext4slower*
f2fsslower*
filegone*
filelife*
fileslower*
filetop*
funccount*
funcinterval*
funclatency*
funcslower*
gethostlatency*
hardirqs*
inject*
javacalls*
javaflow*
javagc*
javaobjnew*
javastat*
javathreads*
killsnoop*
klockstat*
kvmexit*
lib/
llcstat*
mdflush*
memleak*
mountsnoop*
mysqld_qslower*
netqtop*
netqtop.c
nfsdist*
nfsslower*
nodegc*
nodestat*
offcputime*
offwaketime*
old/
oomkill*
opensnoop*
perlcalls*
perlflow*
perlstat*
phpcalls*
phpflow*
phpstat*
pidpersec*
ppchcalls*
profile*
pythoncalls*
pythonflow*
pythongc*
pythonstat*
rdmaucma*
readahead*
reset-trace*
rubycalls*
rubyflow*
rubygc*
rubyobjnew*
rubystat*
runqlat*
runqlen*
runqslower*
shmsnoop*
slabratetop*
sofdsnoop*
softirqs*
solisten*
sslsniff*
stackcount*
statsnoop*
swapin*
syncsnoop*
syscount*
tclcalls*
tclflow*
tclobjnew*
tclstat*
tcpaccept*
tcpcong*
tcpconnect*
tcpconnlat*
tcpdrop*
tcplife*
tcpretrans*
tcprtt*
tcpstates*
tcpsubnet*
tcpsynbl*
tcptop*
tcptracer*
threadsnoop*
tplist*
trace*
ttysnoop*
vfscount*
vfsstat*
virtiostat*
wakeuptime*
wqlat*
xfsdist*
xfsslower*
zfsdist*
zfsslower*
```

the .c seems garbage, but all of these technically could also be in PATH. though i guess nobody else does that either so this is fine

Reply:
ah. all of these (python/bash) tools were rewritten in C and are present in libbpf-tools/ subfolder with a separate makefile. some distros package those (with a bpf- prefix). i guess i could do that, maybe in a separate template